### PR TITLE
Update .desktop file

### PR DESCRIPTION
**Pull request by jn64**
Please refer to commit messages for details.

### CODE REVIEW

File: TS/treesheets.desktop [hunk 1]
```diff
@@ -2,12 +2,13 @@
 Version=1.0
 Type=Application
 Name=TreeSheets
-GenericName=Treesheets
-Comment=A hierarhical spreadsheet / outliner productivity tool.
+GenericName=Spreadsheet
```
Comment (aardappel) [inline TS/treesheets.desktop:5]: Why is this changed to Spreadsheet? It isn't a spreadsheet..

Comment (jn64) [reply TS/treesheets.desktop:5]: What would you suggest? I based it on your own description:

> Free Form Data Organizer
> (Hierarchical Spreadsheet)

and

> A "hierarchical spreadsheet" that is a great replacement for spreadsheets, mind mappers, outliners, PIMs, text editors and small databases.

Comment (aardappel) [reply TS/treesheets.desktop:5]: either of those is fine..

Comment (jn64) [reply TS/treesheets.desktop]: Ok I changed it to Hierarchical Spreadsheet.

**Comment by aardappel** [comment]
Thanks!

**Comment by aardappel** [comment]
@jn64 note https://github.com/aardappel/treesheets/commit/c4cae825e407781f16055081105c321b7ee6cb29
This PR caused:
```
/home/travis/build/aardappel/treesheets/appdir/treesheets.desktop: error: value "Office;Utility;Spreadsheet;TextEditor" for string list key "Categories" in group "Desktop Entry" does not have a semicolon (';') as trailing character
ERROR: Desktop file contains errors. Please fix them. Please see
       https://standards.freedesktop.org/desktop-entry-spec/1.0/n       for more information.
```
